test/cli/helloworld_test.py

```diff
@@ -177,17 +190,30 @@ def test_vs_project_absolute_path():
     assert __getVsConfigs(stdout, filename) == 'Debug|Win32 Debug|x64 Release|Win32 Release|x64'
     assert stderr == '[%s:5]: (error) Division by zero.\n' % filename
 
-def test_cppcheck_project_local_path():
+def __test_cppcheck_project_local_path(extra_args=None, exp_vs_cfg='Debug|x64'):
     args = [
         '--template=cppcheck1',
         '--platform=win64',
         '--project=helloworld.cppcheck'
     ]
+    if extra_args:
+        args += extra_args
     ret, stdout, stderr = cppcheck(args, cwd=__proj_dir)
     assert ret == 0, stdout
-    assert __getVsConfigs(stdout, 'main.c') == 'Debug|x64'
+    assert __getVsConfigs(stdout, 'main.c') == exp_vs_cfg
     assert stderr == '[main.c:5]: (error) Division by zero.\n'
 
+def test_cppcheck_project_local_path():
+    __test_cppcheck_project_local_path()
+
+@pytest.mark.xfail  # TODO: no source files found
+def test_cppcheck_project_local_path_select_one():
+    __test_cppcheck_project_local_path(['--project-configuration=Release|Win32'], 'Release|Win32')
+
+@pytest.mark.xfail  # TODO: no source files found
+def test_cppcheck_project_local_path_select_one_multiple():
+    __test_cppcheck_project_local_path(['--project-configuration=Debug|Win32', '--project-configuration=Release|Win32'], 'Release|Win32')
```

Comment:
I have not looked into why it does not find source files. I will file a separate ticket about that later.

Part of this PR is needed for some other changes I am currently working on.